build/build-image/rsyncd.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file will set up and run rsyncd in order to sync kubernetes sources back
```

Comment:
can you add a little more info like "in order to sync kubernetes source between the local filesystem and the docker-build-data container"

I see this sentence and I ask myself, 'sync source back and forth between what and what?'